src/plexil-adapter/OwExecutive.cpp

```diff
@@ -51,6 +50,14 @@ OwExecutive::~OwExecutive()
   if (m_instance) delete m_instance;
 }
 
+//returns true if current plan is finished executing
+bool OwExecutive::getPlanState()
+{
+	bool current_state = PlexilApp->allPlansFinished();
+	return current_state;
```

Comment:
Minor suggestion: could shorten to one line: return PlexilApp->allPlansFinished();

Reply:
Good catch, I fixed it!